drivers/src/nrfx_rtc.c

```diff
@@ -168,7 +168,7 @@ nrfx_err_t nrfx_rtc_cc_disable(nrfx_rtc_t const * const p_instance, uint32_t cha
             return err_code;
         }
     }
-    NRFX_LOG_INFO("RTC id: %d, channel disabled: %d.", p_instance->instance_id, channel);
+    NRFX_LOG_INFO("RTC id: %d, channel disabled: %lu.", p_instance->instance_id, channel);
```

Comment:
channel is 32bit variable so IMO there is no reason to use %lu. %u should be enough to cover uint32_t on 32bit systems. Same story in other lines

Reply:
At least for me, i still get the error when chaning it to %u and not %lu.

```
../../lib/nrfx/drivers/src/nrfx_rtc.c: In function 'nrfx_rtc_cc_disable':
../../lib/nrfx/drivers/src/nrfx_rtc.c:171:19: error: format '%u' expects argument of type 'unsigned int', but argument 3 has type 'uint32_t {aka long unsigned int}' [-Werror=format=]
     NRFX_LOG_INFO("RTC id: %d, channel disabled: %u.", p_instance->instance_id,
                   ^
```

Reply:
Well, okay I see what is happened here. In our case %u should handle 32bit because of our architecture spec, but standard says that unsigned should handle at least 16bit. unsigned long is first type which can handle at least 32bit value. To sum up, %lu is good choice here.